storefront/app/helpers/workarea/storefront/application_helper.rb

```diff
@@ -90,6 +90,32 @@ def optional_field(prompt, *fields, &block)
         content_tag(:div, capture(&block), { class: 'hidden-if-js-enabled',
                                              data: { optional_field: prompt } })
       end
+
+      def image_tag(source, options = {})
+        options = options.symbolize_keys
+        check_for_image_tag_errors(options)
+        skip_pipeline = options.delete(:skip_pipeline)
+        options[:lazy_load] = options[:lazy_load].presence || true
```

Comment:
Unfortunately, 
```
irb(main):001:0> false.present?
=> false
```

So this should be something like `options.key?(:lazy_load) ? options[:lazy_load] : true`


In general for this method though, do we need to copy the implementation from Rails in? Could it be more like:


```
def image_tag(source, options = {})
  options = options.deep_dup
  if options.fetch(:lazy_load, true)
    options[:data] ||= {}
    options[:data][:lazy_image] = resolve_image_source(source, options[:skip_pipeline])
    options[:class] ||= ''
    options[:class] << ' lazy-image'
    source = nil
  end

  super(source, options)
end
```

Reply:
Unfortunately `source` can't be `nil`. You'll get a `Nil location provided. Can't build URI.` error. I'll mess with it.